internal/datacoord/server.go

```diff
@@ -239,7 +239,9 @@ func (s *Server) Register() error {
 	s.session.Register()
 	if s.enableActiveStandBy {
 		s.icSession.ProcessActiveStandBy(nil)
```

Comment:
Why ignore this error? Despite of the fact that nil `activateFunc` will not result error return for now. We shall not depend on the detail implementation of `ProcessActiveStandBy` here